core/src/main/scala/kafka/server/DynamicBrokerConfig.scala

```diff
@@ -920,7 +919,7 @@ class DynamicListenerConfig(server: KafkaBroker) extends BrokerReconfigurable wi
       def immutableListenerConfigs(kafkaConfig: KafkaConfig, prefix: String): Map[String, AnyRef] = {
         kafkaConfig.originalsWithPrefix(prefix, true).asScala.filter { case (key, _) =>
           // skip the reconfigurable configs
-          !DynamicSecurityConfigs.contains(key) && !SocketServer.ListenerReconfigurableConfigs.contains(key)
+          !DynamicSecurityConfigs.contains(key) && !SocketServer.ListenerReconfigurableConfigs.contains(key) && !DataPlaneAcceptor.ListenerReconfigurableConfigs.contains(key)
```

Comment:
IIUC the reconfigurability of thread numbers is not specific to the data plane, so why is the constant declared on `DataPlaneAcceptor` rather than `Acceptor`?

Reply:
Configuring the number of processor threads is unique to the data plane. The control place always has a single processor to guarantee controller requests are handled in order.

Reply:
Oh, of course it's single threaded! Thanks.